src/transformers/image_utils.py

```diff
@@ -116,6 +116,20 @@ def to_pil_image(self, image, rescale=None):
             return PIL.Image.fromarray(image)
         return image
 
+    def convert_rgb(self, image):
+        """
+        Converts `PIL.Image.Image` to RGB format. 
+
+        Args:
+            image (`PIL.Image.Image`):
+                The image to convert.
+        """
+        self._ensure_format_supported(image)
+        if not isinstance(image, PIL.Image.Image):
+            return image
+
+        return image.convert("RGB")
+
```

Comment:
As we moved this function from CLIP's feature extractor to image_utils.py, it might make sense to also add the conversion from RGB to BGR to image_utils.py, @hollance.

This ensures that feature extractors can re-use methods defined in the general image_utils.py. Think it would be a good idea to have as much general methods defined there, such that feature extractors don't need to define custom behaviour.

Reply:
@NielsRogge Sure, makes sense. I've done this in the PR for MobileViT.